example/server.js

```diff
@@ -3,21 +3,28 @@
 const express = require('express')
 const falcorExpress = require('falcor-express')
 const Router = require('falcor-router')
-const falcorPostman = require('./index.js')
+const falcorPostman = require('./../falcor-postman.js')
```

Comment:
As that will be entry point, is common practice to name it with the name of the project. (i.e.:, lodash -> lodash.js, ...).